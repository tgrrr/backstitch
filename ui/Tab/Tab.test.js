import React from 'react'
import renderer from 'react-test-renderer'
import { Tab } from '.'

it('Tab: default', () => {
    const component = renderer.create(
        <Tab>
            <>
                Tab
            </>
        </Tab>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
