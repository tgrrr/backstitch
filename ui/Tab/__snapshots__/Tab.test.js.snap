// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Tab "
    data-testid="Tab"
  >
    <button
      className="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="tab"
      tabIndex={-1}
      type="button"
    >
      <span
        className="MuiTab-wrapper"
      />
    </button>
  </div>
</div>
`;
